src/cnc/flavors/aws/ecs/1/build/main_lambda.sh.j2

```diff
@@ -0,0 +1,12 @@
+trap 'send_{{ service.name }}_status_hook $?' EXIT
```

Comment:
As per the comment in `...aws/ecs/1/build/main.sh.j2` I think we can remove this file as it'll conflict with the parent template and cause errors.